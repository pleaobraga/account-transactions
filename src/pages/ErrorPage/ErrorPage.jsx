import React from 'react'
import './ErrorPage.scss'

const ErrorPage = () => {
  return (
    <section className="page page-error">
      <i className="material-icons page-error__icon">error_outline</i>
      <p className="page-error__text">This page could not be loaded</p>
    </section>
  )
}

export default ErrorPage
