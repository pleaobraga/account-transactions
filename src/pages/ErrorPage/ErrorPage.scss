.page {
  &-error {
    flex-direction: column;

    &__icon {
      font-size: 20rem;
      color: red;
    }

    &__text {
      font-size: 6rem;
      margin-top: 3rem;
    }
  }
}
