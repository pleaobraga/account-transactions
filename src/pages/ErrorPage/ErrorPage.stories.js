import React from 'react'
import { storiesOf } from '@storybook/react'
import ErrorPage from './ErrorPage'

export default storiesOf('Pages | ErrorPage', module).add(
  'default',
  () => <ErrorPage />,
  { info: { inline: true, header: false } }
)
