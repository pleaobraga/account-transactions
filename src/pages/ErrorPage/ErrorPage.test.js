import React from 'react'
import { shallow } from 'enzyme'
import ErrorPage from './ErrorPage'

describe('ErrorPage', () => {
  const errorPage = shallow(<ErrorPage />)

  it('render proprely', () => {
    expect(errorPage).toMatchSnapshot()
  })
})
