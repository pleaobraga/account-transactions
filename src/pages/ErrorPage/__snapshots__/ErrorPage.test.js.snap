// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage render proprely 1`] = `
<section
  className="page page-error"
>
  <i
    className="material-icons page-error__icon"
  >
    error_outline
  </i>
  <p
    className="page-error__text"
  >
    This page could not be loaded
  </p>
</section>
`;
